openapi/components/schemas/GatewayAccounts/FinancialSettings.yaml

```
@@ -22,6 +22,8 @@ properties:
         bips:
           $ref: ../Bips.yaml
         period:
-          $ref: RiskReservePeriod.yaml
+          description: Instruction for the calculation of reserve release time.
```

nitpick: suggestion to use the same structure as the period anchor description 
```suggestion
          description: Instruction for calculating the reserve release time.
```